bedrock/mozorg/templates/mozorg/home/home-b.html

```diff
@@ -0,0 +1,69 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/home/home.html" %}
+
+{% block page_css %}
+  {% stylesheet 'home-b' %}
+{% endblock %}
+
+{% block intro %}
+<div class="content" id="intro-wrapper">
+  <div id="intro-mozilla">
+    <div class="intro-content">
+      <h2>
+        {{ _('Internet for people, <br>not profit.') }}
+      </h2>
+      <h3>
+      {% trans %}
+        Hi. We’re Mozilla, the proudly non-profit champions of the Internet,
+        helping to keep it healthy, open and accessible to all.
+      {% endtrans %}
+      </h3>
+    </div>{#--/.intro-content--#}
+  </div>{#--/#intro-mozilla-_#}
+
+  <div id="intro-promo">
+    <div id="intro-promo-content-wrapper" class="state-default">
+      <div class="intro-content">
+        {# different promos based on browser/FxA state #}
+
+        <div class="show-not-fx show-default">
+          <h2>{{ _('Unleash your Internet') }}</h2>
+          <p>{{ _('Set the Web free and your mind will follow.') }}</p>
+
+          {{ download_firefox(dom_id='download-intro', button_color='button-arrow', download_location='primary cta') }}
+        </div>{#--/.show-default --#}
+
+        {# shown for Fx desktop signed out of sync, and Fx for Android & iOS #}
+        <div class="show-fx-signed-out show-fx-mobile">
+          <h2>{{ _('Your web, how you like it, everywhere') }}</h2>
+
+          <p>{{ _('With your Firefox Account you can sync your bookmarks, passwords, open tabs and more, and access them everywhere you use Firefox.') }}</p>
+
+          {# for desktop users, link below is wired to open about:accounts #}
+          <a href="{{ url('firefox.features.sync') }}" class="button" id="fxa-sign-in">{{ _('Get started with Sync') }}</a>
+        </div>{#--/.show-fx-signed-out--#}
+
+        <div class="show-fx-signed-in">
+          <h2>Pocket for Firefox</h2>
```

Comment:
Unwrapped strings here and below?

Reply:
It's en-US only, and these strings are not set to be localized, so I don't see a reason to wrap them.

Reply:
Up to you, but I've fallen for this way too many times now in my own PRs